docs/parse.md

```diff
@@ -0,0 +1,47 @@
+# `Temporal.parse`
+
+The `Temporal.parse` function parses a full ISO string and divides it into its
+respective components, which can then be used to produce the corresponding
+Temporal objects.
+
+This function can be used to parse full valid ISO strings that represent invalid
+absolutes and salvage valid components from the string.
+
+## Syntax
+
+```javascript
+const parseResult = Temporal.parse(input);
+```
+
+### Parameters
+
+- `input` (string): The input string to be parsed. It should be a full valid
+ISO string, but it may or may not represent a valid absolute.
+
+### Returns
+
+The returned value is an object that contains all the components of the parsed
+ISO string. It has a structure described by the following TypeScript interface:
+
+```typescript
+interface parseResult {
+    absolute: string;
+    dateTime: string;
+    date: string;
+    time: string;
+    yearMonth: string;
+    monthDay: string;
+    zone: {
+        ianaName?: string;
+        offset?: string;
+    }
```

Comment:
I have a bunch of concerns with the `zone` type here:
* there's no other Temporal API that accepts this shape. I worry that users would logically assume they could construct a TimeZone instance using this object, but they can't.
*  no other Temporal API calls it `zone`... it's always `timeZone`
* `ianaName` is called `timeZoneIdentifier` [elsewhere](https://tc39.es/proposal-temporal/docs/timezone.html#new-Temporal-TimeZone). Many users won't know what IANA is.
* `offset`'s name, unlike elsewhere in Temporal  (e.g. `TimeZone.prototype.getOffsetStringFor` vs. `TimeZone.prototype.getOffsetNanosecondsFor`), doesn't clarify whether it's a string or a number, or if numeric what unit it's using.
* IMHO it's a little weird to have only one nested type. Why nest `ianaName` under zone, but not day under `monthDay`?  Not suggesting we do this this, just pointing out the inconsistency. 

My suggestion would be to instead have two top-level properties, e.g. `timeZoneIdentifier` and `timeZoneOffset` (or maybe `timeZoneOffsetString`, although that may be implicit if all fields are strings?)